src/main/java/create_task_use_case/CreateTaskInputData.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * Create Task Input Data Class.
+ * Author: Fardin Faruk
+ * Modified By: Fardin Faruk
+ * Created: Nov 19, 2022
+ * Last Modified: Nov 27, 2022
+ */
+package create_task_use_case;
+
+public class CreateTaskInputData {
+
+    private String title;
+    private boolean completed;
+    private int dayIndex;
+
+    public CreateTaskInputData(String title, int dayIndex){
```

Comment:
Please provide a set of input values and ill implement a switch for conversion.

Reply:
You can refer to my reply in the controller class, but as the view does not deal directly with the input data, we don't need to worry about compatibility with the UI branch and your branch. Since the controller will be given a String, the input data should take in that String and convert it to an int that can be used in the interactor.

Reply:
Please check out my latest push regarding switch.